syntax = "proto3";

package S2S;



message DBPlayerData
{
	DBPlayerBaseInfo player_base_info = 1;
}

message DBPlayerBaseInfo
{
	int64 uid = 1; 
	int64 openid = 2; 
	string nickname = 3; 
	string ip = 4;
}






